app/javascript/packages/address-search/components/full-address-search.spec.tsx

```diff
@@ -21,9 +71,9 @@ describe('FullAddressSearch', () => {
             usStatesTerritories={usStatesTerritories}
             onFoundLocations={handleLocationsFound}
             locationsURL={locationsURL}
```

Comment:
I created interface FullAddressSearchProps and as a result had to update these tests to satisfy the data type- that is the reason for the following updates in this test file